android/routes/ui/src/main/java/com/trafi/routes/ui/RoutesScreen.kt

```diff
@@ -143,34 +145,41 @@ fun RoutesScreen(
                 RouteSearchBody(
                     state = routesViewModel.state,
                     onRouteClick = onRouteClick,
-                    modifier = Modifier.padding(top = Spacing.md)
+                    modifier = Modifier.padding(top = Spacing.md),
                 )
             }
         }
     }
 }
 
 @Composable
-private fun ColumnScope.RouteSearchBody(
+private fun RouteSearchBody(
     state: RoutesResultState,
     onRouteClick: (Route) -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    when (state) {
-        RoutesResultState.NoResults -> {
-            Text(
-                "No results",
-                modifier = modifier.align(Alignment.CenterHorizontally),
-                style = MaasTheme.typography.textL
-            )
-        }
-        RoutesResultState.Loading -> {
-            CircularProgressIndicator(
-                modifier = modifier.align(Alignment.CenterHorizontally)
-            )
+    Column {
```

Comment:
Since `RouteSearchBody` is already called inside of a `Column` (and hence why it was an extension on `ColumnScope`), there should be no need to wrap it in another `Column` - just add the `Divider` as you did below. If a `Column` is necessary, then the root composable should respect the `modifier` parameter.